score-lang-compiler/src/main/java/org/openscore/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -111,25 +111,34 @@ public Executable transformToExecutable(ParsedSlang parsedSlang, String execName
                 if(!executableRawData.containsKey(SlangTextualKeys.WORKFLOW_KEY)){
                     throw new RuntimeException("Error compiling " + parsedSlang.getName() + ". Flow: " + execName + " has no workflow property");
                 }
-                LinkedHashMap<String, Map<String, Object>> workFlowRawData;
+                List<Map<String, Map<String, Object>>> workFlowRawData;
```

Comment:
I don't understand your comment...
The workFlow data is now a list of maps, not a map, and I need to look in the list for the map with the on_failure key, and also remove it from the list (which I do with the iterator). Do you have an example of how I can improve the code?